docs/man/rpm-dependency-generators.7.scd

```diff
@@ -0,0 +1,387 @@
+RPM-DEPENDENCY-GENERATORS(7)
+
+# NAME
+*rpm-dependency-generators* - RPM file attribute and dependency generator
+interface
```

Comment:
Hmm. There may be arbitrary number of generators, but the dependency generator interface that this manual describes is singular. Makes me wonder if we should drop the plural from the man page name. OTOH we have "rpm-macros" describing the macro processor :thinking: